src/main/python/hydra/lib/hydrabase.py

```diff
@@ -293,28 +294,30 @@ def create_hydra_app(self, name, app_path, app_args, cpus, mem, ports=None, cons
         self.refresh_app_info(name)
         return r
 
-    def create_app_group(self, name, group_name, apps_in_group, analyser):
+    def create_app_instances_group(self, app_name, group_name, num_app_instances, analyser):
```

Comment:
@tahir24434 
Along with this change add the following to keep backward compatibility, Till we can clean out all the stuff in kraken.

def create_app_group(self, **kwargs):
       return self.create_app_instance_group(**kwargs)

After all the cleanup across projects we will remove the old api. 

I would suggest following this strategy for any api that is being used across projects.